addons/hr_recruitment_skills/models/hr_candidate.py

```diff
@@ -35,6 +35,9 @@ def _compute_skill_ids(self):
             candidate.skill_ids = candidate.candidate_skill_ids.skill_id
 
     def _update_employee_from_candidate(self):
+        if self.env.context.get('creating_employee'):
+            return super()._update_employee_from_candidate()
+
         vals_list = []
         for candidate in self:
             existing_skills = candidate.employee_id.employee_skill_ids.skill_id
```

Comment:
To be discussed, but this entire method seems useless IMO :thinking: It is only triggered once `employee_id` is updated on the candidate, and the only process we currently have to change that field is to create the employee... which means the update of the employee is only triggered upon creation of the record, serving no purpose :sweat_smile: if it was triggered upon updating a specific set of fields, why not, otherwise we could just drop that method altogether.